.github/workflows/build.yml

```
@@ -42,8 +42,8 @@ jobs:
         with:
           validate-wrappers: true
 
-      - name: Run All Tests
-        run: ./gradlew check
+      - name: Run All Tests and verify :sample module
+        run: ./gradlew check :sample:verifyPaparazziDebug
```

This can be splitted in 2 distinct jobs if you prefer.

---

Should this be left intact and instead hook :check to depend on :sample:verifyPaparazziDebug? That way local `gradlew build` would also verify them.

---

That is another alternative indeed.  
Although since Paparazzi does not do this by default, it might be misleading for readers to configure it this way.

---

yes please add this dependency configuration to sample.gradle. Thanks for adding this!